sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/FullFormsApiResponseDecoder.kt

```diff
@@ -0,0 +1,26 @@
+package com.klaviyo.analytics.networking.requests
+
+import org.json.JSONObject
+
+object FullFormsApiResponseDecoder {
+
+    /**
+     * We'll be updating the endpoint to only return one form for mobile clients
+     * but for now, this uses the web version and only grabs the top form
+     */
+    fun onlyFirstForm(json: JSONObject): JSONObject =
+        JSONObject().apply {
+            put(
+                FullsFormsResponse.FULL_FORMS,
+                json.getJSONArray(FullsFormsResponse.FULL_FORMS)[0]
+            )
+            put(
+                FullsFormsResponse.FORM_SETTINGS,
+                json.getJSONObject(FullsFormsResponse.FORM_SETTINGS)
+            )
+            put(
+                FullsFormsResponse.DYNAMIC_INFO_CONFIG,
+                json.getJSONObject(FullsFormsResponse.DYNAMIC_INFO_CONFIG)
+            )
```

Comment:
(question): Are these global settings/config info or do we need to key into them somehow to get details for the specific form we want to display?

Reply:
I'm pretty certain we don't need to key into these. The three fields here are what we need to send as args to the 'loadForm' Klaviyo.js call, all this decoder really does it take the top of the list of full forms sent. I'm not further serializing anything to bc we're really just relaying it all forward.